security/keycloak-jwt/src/test/java/io/quarkus/ts/openshift/security/keycloak/containers/KeycloakQuarkusTestResource.java

```diff
@@ -0,0 +1,57 @@
+package io.quarkus.ts.openshift.security.keycloak.containers;
+
+import io.quarkus.test.common.QuarkusTestResourceLifecycleManager;
+import org.testcontainers.containers.BindMode;
+import org.testcontainers.containers.GenericContainer;
+import org.testcontainers.containers.wait.strategy.Wait;
+
+import java.time.Duration;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Optional;
+
+public class KeycloakQuarkusTestResource implements QuarkusTestResourceLifecycleManager {
```

Comment:
Don't we have this duplicated? Perhaps move to the `common` module.

Reply:
For me, It's ok to move this resource into the `common` module. Are you ok with this, @rsvoboda ?

Reply:
@Ladicek and @rsvoboda I've moved the Keycloak test resource into the common module. Let me know your thoughts about the current approach.